### PR TITLE
Bug/duplicates

### PR DESCRIPTION
**Pull request by burzum**
https://github.com/UseMuffin/Tags/issues/24

### CODE REVIEW

**Comment by burzum** [comment]
Has anyone an idea why the fixtures fail? They work locally for me but Travis doesn't like them... I don't want to make a thousand trial and error commits just to satisfy Travis...

File: config/Migrations/20151013144821_unique_tags.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        $table = $this->table('tags_tags');
+
+        $table->addColumn('tag_key', 'string', [
+            'default' => null,
+            'limit' => 255,
+            'null' => false,
+        ]);
+
+        $table->addIndex(['tag_key', 'label', 'namespace'], ['unique' => true]);
+        $table->save();
```
Comment (jadb) [inline config/Migrations/20151013144821_unique_tags.php]: Using `save()` but comment in docblock suggests against using it?

Comment (burzum) [inline config/Migrations/20151013144821_unique_tags.php]: I'll change it, gimme a minute.

**Comment by jadb** [comment]
Looks like you fixed one occurrence but not the other :wink:

**Comment by burzum** [comment]
True, also I think update() is more correct than create() here.

**Comment by jadb** [comment]
:+1:

**Comment by burzum** [comment]
Thank you!

**Pull request merged**